src/aatoolbox/config/countryconfig.py

```diff
@@ -26,33 +26,160 @@ class CodABConfig(BaseModel):
+        if "{admin_level}" not in layer_base_name:
+            raise ValueError(
+                "The layer base name must contain "
+                "an {admin_level} placeholder."
+            )
+        return layer_base_name
+
+
+class FewsNetConfig(BaseModel):
+    """FEWS NET configuration.
+
+    Parameters
+    ----------
+    region_name: str
+        Name of the region the country belongs to. Needed to download the
+        regional FEWS NET data
+    """
+
+    # values dictionary gets build in order attributes are listed
+    # so first define the dict before the region_name
+    region_name_code_mapping: Dict[str, str] = {
+        "caribbean-central-america": "LAC",
+        "central-asia": "CA",
+        "east-africa": "EA",
+        "southern-africa": "SA",
+        "west-africa": "WA",
+    }
+    region_name: str
+
+    @validator("region_name")
+    def regionname_valid(cls, v, values):
+        """Check that regionname is one of the valid ones."""
+        valid_regionnames = values["region_name_code_mapping"].keys()
+        if v not in valid_regionnames:
+            raise ValueError(
+                f"Invalid region name: {v}. "
+                f"Should be one of {', '.join(valid_regionnames)}"
+            )
+        return v
+
+    @root_validator(pre=False, skip_on_failure=True)
+    def _set_region_code(cls, values) -> dict:
+        """Set region code based on region name."""
+        values["region_code"] = values["region_name_code_mapping"][
+            values["region_name"]
+        ]
+        return values
+
+
+class UsgsNdviConfig(BaseModel):
+    """USGS NDVI configuration.
+
+    Parameters
+    ----------
+    area_name : str
+        Name of the USGS NDVI coverage area the country belongs to.
+        Needed to download the regional NDVI data.
+    """
+
+    area_name_mapping: Dict[str, Tuple[str, str]] = {
+        "north-africa": ("africa/north", "na"),
+        "east-africa": ("africa/east", "ea"),
+        "southern-africa": ("africa/southern", "sa"),
+        "west-africa": ("africa/west", "wa"),
+        "central-asia": ("asia/centralasia", "cta"),
+        "yemen": ("asia/middleeast/yemen", "yem"),
+        "central-america": ("lac/camcar/centralamerica", "ca"),
+        "hispaniola": ("lac/camcar/caribbean/hispaniola", "hi"),
+    }
+    area_name: str
+
+    @validator("area_name")
+    def area_name_valid(cls, v, values) -> str:
+        """Check that area_name is valid."""
+        valid_area_names = values["area_name_mapping"].keys()
+        if v not in valid_area_names:
+            raise ValueError(
+                f"Invalid area name: {v}. "
+                f"Should be one of {', '.join(valid_area_names)}"
+            )
+        return v
+
+    @root_validator(pre=False, skip_on_failure=True)
+    def _set_area_codes(cls, values) -> dict:
+        """Set NDVI url and prefix from area."""
+        area_values = values["area_name_mapping"]["area_name"]
```

Comment:
getting an error here. Should be `area_values = values["area_name_mapping"][values["area_name"]]